### PR TITLE
add Vector class init directly with GeoDataFrame

### PR DESCRIPTION
**Pull request by rhugonnet**
Just a couple lines, tested.

### CODE REVIEW

File: GeoUtils/vector_tools.py [hunk 1]
```diff
@@ -25,9 +25,15 @@ def __init__(self, filename):
         :return: A Vector object
         """
```
Comment (adehecq) [inline GeoUtils/vector_tools.py:27]: Need to update docstring

File: GeoUtils/vector_tools.py [hunk 2]
```diff
@@ -25,9 +25,15 @@ def __init__(self, filename):
...
-        self.name = filename
+        if isinstance(filename,str):
+            ds = gpd.read_file(filename)
+            self.ds = ds
+            self.name = filename
+        elif isinstance(filename,gpd.GeoDataFrame):
+            self.ds = filename
+            self.name = None
+        else:
+            raise ValueError('filename argument not recognised.')
```
Comment (iamdonovan) [inline GeoUtils/vector_tools.py:36]: what kind of spelling is this?

Comment (adehecq) [reply GeoUtils/vector_tools.py:36]: I think it's an old spelling from a country that once strove for independence !

Comment (rhugonnet) [reply GeoUtils/vector_tools.py:36]: for my defense I copied/pasted this from raster_tools :p

**Review by adehecq — APPROVED** [review]

File: GeoUtils/vector_tools.py (hunk 1 above)
Comment (rhugonnet) [reply GeoUtils/vector_tools.py:27]: done

**Review by atedstone — APPROVED** [review]

**Pull request merged**